src/themes/shared/typography.js

```diff
@@ -5,119 +5,123 @@ This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 */
 // @flow
-import type {FontTokensT, TypographyT} from '../types.js';
+import type {TypographyT, FontTokensT} from '../types.js';
 
-export const fontTokens: FontTokensT = {
+export const defaultFontTokens: FontTokensT = {
   primaryFontFamily:
     'system-ui, "Helvetica Neue", Helvetica, Arial, sans-serif',
 };
 
-// color constants
-export default (themePrimitives: FontTokensT = fontTokens): TypographyT => {
+export default (
+  customFontTokens?: $Shape<FontTokensT> = defaultFontTokens,
+): TypographyT => {
+  const {
+    primaryFontFamily = defaultFontTokens.primaryFontFamily,
```

Comment:
Could we avoid assigning the default in two places (16 and 19)?

Reply:
I was trying to update the function so it could accept nothing, a partial font tokens object, or a full font tokens object. Right now it only accepts nothing or a full font tokens object, which is why we need to call `getTypography` differently depending on whether or not `primaryFontFamily` is defined. I thought it might be nice to allow the font tokens to also be partially provided going forward. but i'll take it out and if we add new font tokens we can worry about it then.